orchestra/cmds/check_branch.py

```diff
@@ -0,0 +1,87 @@
+import itertools
```

Comment:
No copyright notice, does this pass check-branch? :P

Reply:
Uh, most files in orchestra don't have copyright notices